test/jdk/java/awt/print/PrinterJob/ImagePrinting/NullClipARGB.java

```diff
@@ -26,27 +26,36 @@
  * @key printer
  * @bug 8061392
  * @summary Test no NPE when printing transparency with null clip.
+ * @library /test/lib
  */
 
 import java.awt.*;
 import java.awt.image.*;
 import java.awt.print.*;
+import javax.print.DocFlavor;
+import javax.print.PrintService;
+import javax.print.PrintServiceLookup;
+import jtreg.SkippedException;
 
 public class NullClipARGB implements Printable {
 
     public static void main( String[] args ) {
+        PrintService[] svc = PrintServiceLookup.lookupPrintServices(DocFlavor.SERVICE_FORMATTED.PRINTABLE, null);
```

Comment:
Do we need to ask for `DocFlavor.SERVICE_FORMATTED.PRINTABLE` here? Can we just ask for `null`?

Reply:
Correct me if I'm wrong, but I think if we ask for null here it may find a PrintService that does not support PRINTABLE flavor so SkippedException will not be thrown because svc.lenght will be greater than 0 and we may end up with an exception thrown later in pj.print().

Reply:
Oh, OK. So the idea is that if we use `setPrintable`, we need to lookup the service that accepts `.PRINTABLE`?

Reply:
This is my understanding.